README.md

```diff
@@ -264,6 +265,39 @@ You may choose to either start from the basics of NLP and gradually move to deep
 
+Machine Learning today is one of the most sought-after skills in the market. A lot of Software Engineers are picking up ML, simply because it is a highly paid skill.
+So, how do you go about Machine Learning?
 
-Topics to come: Machine Learning, Deep Learning, Computer Vision, Cybersecurity and maybe HCI  
+•	First things first - the prerequisites:
+
+o	Basic calculus. In Machine Learning, you’d be working on a lot of optimizations that require knowledge of Calculus. It would be highly recommended that you are aware of functions, limits, differentiation, maxima, minima, etc.
+o	Linear Algebra. When you talk about ML, you will be dealing with matrices and vectors every day. So, knowledge of Linear Algebra is a must. However, you’d also be required to know about other important topics like Eigenvalues and Eigenvectors.
+o	Probability. Most ML algorithms try to “model” the underlying phenomena that generated the observed data. All of this modelling is probabilistic. It is therefore highly recommended that you are comfortable with the theory of Probability.
+
+•	Getting into actual ML:
+
+#### Courses
+
+1.	[Machine Learning]( https://www.coursera.org/learn/machine-learning) by Andrew Ng. on Coursera. It is the best course there is, on this topic, and takes you slowly and efficiently through all the different algorithms like, Regression, Classification, to name a few. Once you finish this about 8-week long course, you should be ready to start taking up small ML projects of your own for practice.
+
+2.	[Machine Learning Crash Course](https://developers.google.com/machine-learning/crash-course/ml-intro) as provided by Google, is also a good way to step into the world of ML. However, it is, as its name suggests, a crash course, and would not go as into depth as the Coursera one does. So, if you’re in a hurry, try this one out, and you’ll be done in almost a week or two!
+
+
+#### Books
+
+For those of you, who prefer to learn & understand through a good, thorough read:
+
+1.	[Bayesian Reasoning and Machine Learning](http://web4.cs.ucl.ac.uk/staff/D.Barber/textbook/091117.pdf) by David Barber. 
+Speaking of Bayesian statistics, this one is a classic. This takes a Bayesian statistics approach to machine learning. A book worth checking out for anyone getting into the machine learning field.
+
+2.	[Understanding Machine Learning](http://www.cs.huji.ac.il/~shais/UnderstandingMachineLearning/index.html) by Shai Shalev-Shwartz and Shai Ben-David. This book gives a structured introduction to machine learning. It looks at the fundamental theories of machine learning and the mathematical derivations that transform these concepts into practical algorithms. Following that, it covers a list of ML algorithms, including (but not limited to), stochastic gradient descent, neural networks, and structured output learning.
+
+
```

Comment:
Lets not forget PRML by Ian Bishop [Best ML book] and ML by Tom Mitchell

Reply:
ISLR is another classic and there are python implementations here: https://github.com/JWarmenhoven/ISLR-python